Content.Client/HealthAnalyzer/UI/HealthAnalyzerWindow.xaml.cs

```diff
@@ -110,18 +110,30 @@ public void Populate(HealthAnalyzerScannedUserMessage msg)
 
             // Alerts
 
-            AlertsDivider.Visible = msg.Bleeding == true;
-            AlertsContainer.Visible = msg.Bleeding == true;
+            var showAlerts = msg.Unrevivable == true || msg.Bleeding == true;
 
-            if (msg.Bleeding == true)
-            {
+            AlertsDivider.Visible = showAlerts;
+            AlertsContainer.Visible = showAlerts;
+
+            if (showAlerts)
+                // Wouldn't this end up with these being stuck on after bleeding/etc stops?
```

Comment:
It's not an issue because the alerts container gets hidden when there is no more bleeding

Reply:
Yup, see now, derp